app/actions/__tests__/searchActions.spec.js

```diff
@@ -1,8 +1,8 @@
 import { expect } from 'chai';
 import simple from 'simple-mock';
 
-import searchActions from 'actions/searchActions';
-import apiUtils from 'utils/APIUtils';
+import { searchResources } from 'actions/searchActions';
+import * as apiUtils from 'utils/APIUtils';
```

Comment:
Actions and utils are exported using named exports so they can be imported with 
```javascript
import { searchResources } from 'actions/searchActions';
```
both in tests and in regular files. 

The `import * as apiUtils from 'utils/APIUtils';` is used because otherwise there is no way to mock a single method from the utils.

Exporting only the whole utils or actions module might be better but that would have required quite extensive changes in every file that uses any of the actions or util methods. That's why I didn't do it yet.